autocomplete/cached_cloud_client.go

```diff
@@ -137,3 +138,33 @@ func (ccc *cachedCloudClient) ListProjects(ctx context.Context, orgName string)
 	_ = saveJSON(ccc.installationName, filename, &cached)
 	return projects, nil
 }
+
+func (ccc *cachedCloudClient) ListSatellites(ctx context.Context, orgName string, includeHidden bool) ([]cloud.SatelliteInstance, error) {
+	cached := struct {
+		Org           string
+		IncludeHidden bool
+		Satellites    []string `json:"satellites"`
+	}{}
+	filename := ".autocomplete.satellites"
+	if err := readJSON(ccc.installationName, filename, &cached); err == nil && cached.Org == orgName && cached.IncludeHidden == includeHidden {
+		res := []cloud.SatelliteInstance{}
+		for _, s := range cached.Satellites {
+			res = append(res, cloud.SatelliteInstance{
+				Name: s,
+			})
+		}
+		return res, nil
+	}
+	cached.Satellites = nil
```

Comment:
This seems redundant, I think. I can't see a way for `cached.Satellites` to not already be nil.

Reply:
it actually isn't due to the `err == nil && cached.Org == orgName && cached.IncludeHidden == includeHidden {`.

fortunately the unit tests caught that when I implemented `ListProjects` previously.